vendor/cicicom/s-lg-i3.yaml

```
@@ -0,0 +1,67 @@
+name: SLG-I3 Parking Sensor
+description: Cicicom's LoRaWAN smart parking sensor SL-G-I3 accurately detects parking events, by utilizing cutting-edge triple detection technology (magnetic sensor, motion sensor, radar). All sensors and algorithms are tuned and optimized for ultra-low power consumption, providing continuous vehicle detection with higher than 99% accuracy and resulting to complete monitoring and management of parking spots.
+# Hardware versions (optional, use when you have revisions)
+hardwareVersions:
+  - version: '1.0'
+    numeric: 1
+
+# Firmware versions (at least one is mandatory)
+firmwareVersions:
+  - # Firmware version
+    version: 'V1.4.0'
+    numeric: 14
+    hardwareVersions:
+      - '1.0'
+
+    # LoRaWAN Device Profiles per region
+    # Supported regions are EU863-870, US902-928, AU915-928, AS923, CN779-787, EU433, CN470-510, KR920-923, IN865-867, RU864-870
+    profiles:
+      EU863-870:
+        # Unique identifier of the profile (lowercase, alphanumeric with dashes, max 36 characters)
+        id: s-lg-i3
```

the id for the profile should point to the filename of the profile
```suggestion
        id: s-lg-i3-profile
```